remail/email_api/service.py

```
@@ -137,20 +137,26 @@ def get_emails(self, date: datetime = None) -> list[Email]:
 
 
 class ImapProtocol(ProtocolTemplate):
-    def __init__(self, email: str, password: str, host: str):
+    def __init__(
+        self, email: str, password: str, host: str, controller: "EmailController"
```

Ruff (0.8.2)

Undefined name `EmailController`

(F821)